components/NewTripForm/index.js

```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+import StyledButton from "../StyledButton";
+import Link from "next/link";
+import { useState } from "react";
+
+export default function NewTripForm({ onSubmit }) {
+  const [startDate, setStartDate] = useState("");
+  const [endDate, setEndDate] = useState("");
+
```

Comment:
You could use just one State here, maybe sth. like this: 

```
const [formState, setFormState] = useState({
    startDate: "",
    endDate: "",
  })
```

Reply:
Thanks for your insight! This is an interesting topic. After reading the React documentation and other related resources, I will leave this code flat for now, but I will keep an eye on it!